crates/revive-strategy/src/cheatcodes/mod.rs

```diff
@@ -349,6 +349,26 @@ impl CheatcodeInspectorStrategyRunner for PvmCheatcodeInspectorStrategyRunner {
                     .unwrap_or(B256::ZERO);
                 Ok(result.abi_encode())
             }
+            t if using_pvm && is::<storeCall>(t) => {
+                tracing::info!(cheatcode = ?cheatcode.as_debug() , using_pvm = ?using_pvm);
+                let &storeCall { target, slot, value } = cheatcode.as_any().downcast_ref().unwrap();
+                if ccx.is_precompile(&target) {
+                    return Err(precompile_error(&target));
+                }
+
+                let target_address_h160 = H160::from_slice(target.as_slice());
+                let _ = execute_with_externalities(|externalities| {
+                    externalities.execute_with(|| {
+                        Pallet::<Runtime>::set_storage(
+                            target_address_h160,
+                            slot.into(),
+                            Some(value.to_vec()),
+                        )
+                    })
+                })
+                .map_err(|_| <&str as Into<Error>>::into("Could not set storage"))?;
+                Ok(Default::default())
```

Comment:
does it mean that we set value in both pallet-revive and REVM (foundry)? If yes I guess it is not needed because after pallet-revive execution state is applied to REVM(foundry) state.

I think that when you set value (vm.store) in REVM(Foundry) you will not have it in pallet-revive - (we do not have contract state migration REVM->pallet-revive). Should we set it in this case in both location?

Reply:
Could you please explain what do you mean?
There are two issues:
1. When we are in PVM we change contract state in REVM and PVM, it should be enough to do it only in PVM as at the end state is applied to REVM
2. When we are in REVM we change contract state only in REVM, which is a problem because we will not have the right state on PVM side - here we are lacking the migration still

How would you like to handle it?

Reply:
> does it mean that we set value in both pallet-revive and REVM (foundry)?

As far as I can tell this will apply the changes only in pallet-revive storage.

Reply:
> > does it mean that we set value in both pallet-revive and REVM (foundry)?
> 
> As far as I can tell this will apply the changes only in pallet-revive storage.

yes, you are right